torch/csrc/distributed/c10d/comm.hpp

```diff
@@ -25,13 +25,15 @@ class TORCH_API GradBucket {
       std::vector<size_t> offsets,
       std::vector<size_t> lengths,
       std::vector<c10::IntArrayRef> sizes_vec,
+      c10::optional<at::Tensor>& global_unique_id,
```

Comment:
Will defer to you, but `global_unique_id` seems like a higher level implementation detail that doesn't fully reflect how users can use this variable. Can we name it something more conceptual / easier to understand?